packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -30,21 +31,16 @@ export default (props) => {
   } = props;
 
   const mainClasses = cx([
-    'demographics-banner',
+    customProps.className,
     'large-demographics-banner',
     { deceased: deceasedDate },
-    customProps.className,
   ]);
 
   delete customProps.className;
 
   return (
     <section className={mainClasses} {...customProps}>
-      {props.photo && (
```

Comment:
By removing this check, this seems non-passive. Again MVB?

Reply:
Added this [Check back](https://github.com/cerner/terra-core/pull/2856/commits/7a71fdbd5a4f8166ac83942fb290090b46c821f4#diff-94aa46c391c8f8d9c9e859a4ae8639af). to keep this changes until Next MVB.